src/components/index/sections/ProblemsToSolveSection.tsx

```diff
@@ -40,30 +46,24 @@ export default function SupportUsSection() {
           {t('index:problems-to-solve-section.heading')}
         </Typography>
       </Grid>
-      <Grid item container>
-        <Grid item xs={12} sm={4}>
-          <div className={classes.problem}>
-            <CommissionsIcon className={classes.icon} />
-            <Typography variant="body2" className={classes.problemLabel}>
-              {t('index:problems-to-solve-section.high-commissions')}
-            </Typography>
-          </div>
+      <Grid item container justify="center" spacing={4}>
+        <Grid item xs={12} sm={4} d-md="flex" className={classes.problem}>
```

Comment:
@ani-kalpachka What's the value of `d-md="flex"`, i think I haven't seen that one in mui. Is that part of bootstrap spec ?